### PR TITLE
add loan page tests

### PR DESCRIPTION
**Pull request by davishooly**
### What does this PR do?
- add test for landing and loan page.

### CODE REVIEW

File: tests/unit/views/personal.spec.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      );
+      expect(wrapperComponent.vm.$data.employer).toBe("muema");
+      expect(wrapperComponent.vm.$data.net).toBe("1000000");
+      expect(wrapperComponent.vm.$data.isPermanentSelected).toBe(false);
+      selectHelper(wrapperComponent, ".container__employed select", 3);
+      wrapperComponent.setData({ nature: "Permanent" });
+      expect(wrapperComponent.vm.$data.isPermanentSelected).toBe(true);
+      const duration = wrapperComponent.findAll(".container__employed select");
+      duration.at(1).trigger("click");
+      duration
```
Comment (munyili-samuel) [inline tests/unit/views/personal.spec.js]: It'd be good to assert that something happens after you `simulate an event`. In this case, it'd be true to assume that `change` leads to something happening hence we can assert it.
This is true for a number of cases ✌️

File: tests/unit/views/landing.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  it("renders loan page ", () => {
+    expect(wrapper.isVueInstance).toBeTruthy();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it("should select a new car", () => {
+    wrapper
+      .findAll(".main__container .button")
+      .at(0)
+      .trigger("click");
```
[the remark this answers is not in the document]
Comment (davishooly) [reply tests/unit/views/landing.spec.js]: cool, let me add assertions to the test cases.

**Comment by davishooly** [comment]
@munyili-samuel I have completed working on the changes you requested.

File: tests/unit/views/personal.spec.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+      expect(wrapperComponent.vm.$data.net).toBe("1000000");
+      expect(wrapperComponent.vm.$data.isPermanentSelected).toBe(false);
+      selectHelper(wrapperComponent, ".container__employed select", 3);
+      wrapperComponent.setData({ nature: "Permanent" });
+      expect(wrapperComponent.vm.$data.isPermanentSelected).toBe(true);
+      const duration = wrapperComponent.findAll(".container__employed select");
+      duration.at(1).trigger("click");
+      duration.at(1).find("option:nth-child(2)").element.selected = true;
+      duration.at(1).trigger("change");
+      expect(wrapperComponent.vm.$data.duration).not.toBe("");
```
Comment (munyili-samuel) [inline tests/unit/views/personal.spec.js]: 😆

**Pull request merged**